managed/services/server/server.go

```diff
@@ -322,106 +345,81 @@ func (s *Server) StartUpdate(ctx context.Context, req *serverpb.StartUpdateReque
 		return nil, status.Error(codes.FailedPrecondition, "Updates are disabled via DISABLE_UPDATES environment variable.")
 	}
 
-	offset, err := s.supervisord.StartUpdate()
```

Comment:
let's have some period for having both types of updates(supervisor and pmm-server-update) because we can't just stop supporting upgrades for big part of our customers suddenly.